src/main.ts

```diff
@@ -57,8 +73,20 @@ if (process.env.NO_CRAWL !== "true") {
     // headless: false,
   });
 
-  // Add first URL to the queue and start the crawl.
-  await crawler.run([config.url]);
+  const isUrlASitemap = config.url.endsWith("sitemap.xml");
```

Comment:
**Suggestion:** Refactor the code to avoid hardcoding the "sitemap.xml" string. You can create a constant for it and use it in the condition.
```suggestion
  const SITEMAP_SUFFIX = "sitemap.xml";
  const isUrlASitemap = config.url.endsWith(SITEMAP_SUFFIX);
```